airbyte-local.sh

```diff
@@ -213,12 +213,26 @@ function validateInput() {
 
 function writeSrcConfig() {
     writeConfig src_config "$tempdir/$src_config_filename"
-    debug "Using source config: $(jq -c < $tempdir/$src_config_filename)"
+    debug "Using source config: $(redactConfigSecrets "$(jq -c < $tempdir/$src_config_filename)" "$(specSrc)")"
```

Comment:
Is it worth wrapping this line in:
```sh
if ((debug)); then
...
fi
```
To prevent the execution of the spec lookup when we are not actually debugging?

Reply:
sure